README.md

```diff
@@ -5,15 +5,17 @@
 
 **Please note: this is not (yet) production-ready code. Proceed with caution.**
 
-- [Introduction](#introduction)
-- [Building the library](#building-the-library)
-- [Using the library](#using-the-library)
-  - [Creating a simple packet](#creating-a-simple-packet)
-  - [Encoding and decoding of packet data](#encoding-and-decoding-of-packet-data)
-  - [Creating a PGP key from raw point data](#creating-a-PGP-key-from-raw-point-data)
+- [PGP packet decoder and encoder](#pgp-packet-decoder-and-encoder)
+  - [Introduction](#introduction)
+  - [Building the library](#building-the-library)
+  - [Using the library](#using-the-library)
+    - [Creating a simple packet](#creating-a-simple-packet)
+    - [Encoding and decoding of packet data](#encoding-and-decoding-of-packet-data)
+    - [Creating a PGP key from raw point data](#creating-a-pgp-key-from-raw-point-data)
 - [Verifying the library](#verifying-the-library)
   - [Clang Tidy](#clang-tidy)
   - [Static analysis using Cppcheck](#static-analysis-using-cppcheck)
+- [Credits](#credits)
 
 ## Introduction
```

Comment:
What do you think of removing the examples from the README and referring just to the examples folder? Keeps it DRY, other projects do this too.